scripts/components/heading-level/heading-level.js

```diff
@@ -46,6 +53,16 @@ const createLevelControl = (targetLevel, selectedLevel, onChange) => {
 	};
 };
 
+/**
+ * A heading level selector.
+ * 
+ * @param {object} props                     - Heading level selector options.
+ * @param {boolean} [props.isCollapsed=true] - If `true`, the control is shown as a button that opens a dropdown menu, otherwise all options are shown inline. 
+ * @param {number} [props.minLevel=1]        - Lowest allowed level.
+ * @param {number} [props.maxLevel=1]        - Highest allowed level.
```

Comment:
Min and max set to 1 seem kinda odd(this is maybe not the place to talk about it but I'm just noticing it) but if this is fine with everyone, all good! 🙂

Reply:
That's definitely a mistake, should be 6, good catch! 😄

Reply:
```suggestion
 * @param {number} [props.minLevel=1]        - Lowest allowed level.
 * @param {number} [props.maxLevel=6]        - Highest allowed level.
```